context_windows.go

```diff
@@ -0,0 +1,9 @@
+package continuity
+
+import "os"
+
+// getUidGidFromFileInfo extracts the user and group IDs from a fileinfo.
+// This is Unix-specific functionality.
```

Comment:
UID/GID are very Unix specific. Windows uses ACES in ACLS (https://msdn.microsoft.com/en-us/library/windows/desktop/aa374872(v=vs.85).aspx).  Certainly as a longer-term plan, this needs to be considered. However, I wish to re-iterate this is just to get this component to compile on Windows, not to functionally prove it. It allows forward progress on containerd.